Features/GoogleAnalytics/functions.php

```diff
@@ -2,18 +2,43 @@
 
 namespace Flynt\Features\GoogleAnalytics;
 
+define(__NAMESPACE__ . '\NS', __NAMESPACE__ . '\\');
+
 require_once __DIR__ . '/GoogleAnalytics.php';
 
 use Flynt\Features\GoogleAnalytics\GoogleAnalytics;
 use Flynt\Utils\Feature;
+use Flynt\Utils\Asset;
 use Flynt\Features\Acf\OptionPages;
 
 add_action('init', 'Flynt\Features\GoogleAnalytics\init', 100);
 
 function init()
 {
     $googleAnalyticsOptions = OptionPages::get('globalOptions', 'feature', 'GoogleAnalytics');
+    $googleAnalyticsOptionsTranslatable = OptionPages::get('translatableOptions', 'feature', 'GoogleAnalytics');
+
     if ($googleAnalyticsOptions) {
         new GoogleAnalytics($googleAnalyticsOptions);
     }
+
+    Asset::enqueue([
+        'type' => 'script',
+        'name' => 'js-cookie',
+        'path' => 'vendor/js-cookie.js'
+    ]);
+
+    Asset::enqueue([
+        'type' => 'script',
+        'name' => 'Flynt/Features/GoogleAnalytics',
+        'path' => 'Features/GoogleAnalytics/script.js',
+        'dependencies' => ['jquery', 'js-cookie']
+    ]);
+
+    $data = [
+        'gaId' => $googleAnalyticsOptions['gaId'],
+        'confirm' => (isset($googleAnalyticsOptionsTranslatable['optOutConfirm'])) ? $googleAnalyticsOptionsTranslatable['optOutConfirm'] : '',
+        'success' => (isset($googleAnalyticsOptionsTranslatable['optOutSuccess'])) ? $googleAnalyticsOptionsTranslatable['optOutSuccess'] : ''
```

Comment:
- these variables are quite wordy and could be shortened
- instead of doing an isset and passing an empty string here, you should just pass the value and check it in the script if it's false before you run the alert (and maybe not run an empty alert in that case, which is weird). Same goes for the confirm box of course.